pkgs/top-level/config.nix

```diff
@@ -151,12 +236,40 @@ in {
         in r // { _undeclared = r; };
     };
 
+  imports = [
+    (mkRemovedOptionModule ["config" "showDerivationWarnings"] "See `config.problems.matchers` and the respective documentation. Be aware that the semantics have changed!")
```

Comment:
Can you reference relevant issues/PRs numbers in that message or even links?

Reply:
I mean you can put anything into the string, although I've never seen any links in there before

Reply:
I did it for bootspec